src/pages/index.js

```diff
@@ -96,142 +103,152 @@ popUpWithImage.setEventListeners();
 //PopUpWithForm
 const popUpEditProfile = new PopUpWithForm(
   "#modal-edit-profile",
-  ({name, about}) => {
+  ({ name, about }) => {
     return new Promise((resolve, reject) => {
```

Comment:
` return new Promise((resolve, reject) => {` is not needed. You can simplify it here


```js
    return api.updateEditProfile({ name, about }).then((updateEditProfile) => {
      userinfo.setUserInfo(updateEditProfile);
    });

```